quiz.css

```diff
@@ -0,0 +1,66 @@
+html {
+	margin: 0;
+	padding: 0;
+}
+
+.productCard {
+	border: 1px solid lightblue;
+	width: 30%;
+	height: 500px;
+	float: left;
+	margin: 5px;
+}
+
+.productTitle {
+	border: 1px dotted lightgrey;
+	text-align: center;
+	font-size: .8em;
+}
+
+.productInfo img {
+	max-width: 100%;
+	max-height: 100%
+}
+
+.productInfo p:first-of-type {
+	font-size: .9em;
+	font-weight: 500;
+	margin: 5px;
+}
```

Comment:
Style Requirement 4 mentioned that the Description text ought to be Justified.

Also, you set the font-size in many of your css selectors. You could probably simplify things by choosing a more generic selector instead of repeating yourself.